### DISCUSSION TITLE
Pole figures

### DISCUSSION THREAD

**Discussion by dharra21** [mtex-toolbox/mtex#1251] [Ask Anything]
I am wondering if anyone can help me out with understanding what my pole figures mean.  Exactly how each of them describe my texture and how you arrive at this conclusion.

I have read so much literature and watched videos on stereographic projections/ pole figures and none the wiser after days of digging. Also if anyone has any lecture notes/  material that helped them understand I would love to learn from them.

My pole figures are below and any thoughts on them would be greatly appreciated.  These are pole figures for the material Titanium /Alpha HCP phase. 

![Screenshot (33)](https://user-images.githubusercontent.com/81887881/136585311-565425da-e1b9-43ea-a777-179c5c0bb2e0.png)

**Comment by kilir** [comment]
Hi,
if you want someone to interpret your results, it may be helpful if you describe your samples, what you did to your samples, how you'd derived these polefigures, how they relate to your specimen and so on.

Cheers,
Rüdiger

**Reply by dharra21** [reply]
Hi Rüdiger

I am new to MTEX and working with MTEX and Dream3d  to process my Titanium EBSD data. My end goal is to detect Micro texture Regions using finite element simulations.  For now I am trying to study my EBSD Titanium sample which is a 2D slice. My original scan was done at a resolution of 3um and pretty coarse, so not the easiest data to work with as I am led to believe.  I am interested in the relationship between the distribution of grain sizes and orientation (still working on that).

These poll figures were generated using Dream3d which looks at the raw data and plots the texture onto 3 poll figures for the HCP phase with respect to each orientation direction <0001), <1010> and <2110>, The red spots correspond to when the texture is strong with respect to the orientation direction (intensity plot). 

I just don't know how to read them, like the spread of the red in the <0001> direction. It looks pretty sheared and I don't know what this suggests.  After hours of searching I am still none the wiser how to read poll figures. Eventually I will be making these in MTEX as well as I progress with understanding coding. I just thought I would I would ask here as I would imagine there are experienced people and this is all new to me. 

Thanks

Dave :)
